python/tests/core/test_feature_view_engine.py

```diff
@@ -32,50 +33,51 @@
 from hsfs.storage_connector import BigQueryConnector, StorageConnector
 
 
-engine.init("python")
-fg1 = feature_group.FeatureGroup(
-    name="test1",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[feature.Feature("id"), feature.Feature("label")],
-    id=11,
-    stream=False,
-)
-
-fg2 = feature_group.FeatureGroup(
-    name="test2",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[feature.Feature("id"), feature.Feature("label")],
-    id=12,
-    stream=False,
-)
-
-fg3 = feature_group.FeatureGroup(
-    name="test2",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[feature.Feature("id"), feature.Feature("label")],
-    id=13,
-    stream=False,
-)
-
-fg4 = feature_group.FeatureGroup(
-    name="test4",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[feature.Feature("id4"), feature.Feature("label4")],
-    id=14,
-    stream=False,
-)
+with mock.patch("hopsworks_common.client.get_instance"):
```

Comment:
Is this relevant to this PR?

Reply:
Yes, since `FeatureGroup.__init__` calls `VariableApi.__init__`.